src/Arcus.Security.Providers.AzureKeyVault/Configuration/KeyVaultOptions.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Arcus.Security.Providers.AzureKeyVault.Configuration
+{
+    /// <summary>
+    /// Represents the available options to configure extra options of the Azure Key Vault for the <see cref="KeyVaultSecretProvider"/>.
+    /// </summary>
+    public class KeyVaultOptions
```

Comment:
Let's use the product name
```suggestion
    public class AzureKeyVaultOptions
```

Reply:
You sure? We also have `KeyVaultConfiguration` and `KeyVaultSecretProvider`?

Reply:
Then it's ok!

Reply:
We should rename those for our next major version though